specs/passenger_spec.rb

```diff
@@ -75,4 +79,48 @@
       end
     end
   end
+
+  describe "total_spend" do
+    before do
+      start_time = Time.parse('2015-05-20T12:14:00+00:00')
+      end_time = start_time + 25 * 60 # 25 minutes
+      trips = [
+        RideShare::Trip.new({cost: 10.00, rating: 3, start_time: start_time, end_time: end_time}),
+        RideShare::Trip.new({cost: 10.00, rating: 3, start_time: start_time, end_time: end_time}),
+        RideShare::Trip.new({cost: 10.00, rating: 3, start_time: start_time, end_time: end_time}),
+        RideShare::Trip.new({cost:nil, rating: 3, start_time: start_time, end_time: nil})
+      ]
+      passenger_data = {
+        id: 7,
+        name: 'Speed Passenger',
+        phone_number: '555.555.5555',
+        trips: trips
+      }
+      @passenger = RideShare::Passenger.new(passenger_data)
+    end
+    it "accurately returns the total for a specific passenger's trips" do
+      @passenger.total_spend.must_equal(30.00)
+    end
+
+    it "does not include unfinished trips in the total spend" do
+      @passenger.total_spend.must_equal(30.00)
+    end
+  end
+
+  describe "total_ride_time_minutes" do
```

Comment:
What about drivers with no trips, and several trips, make sure the addition is done properly.